src/lib/pages/Profile.svelte

```diff
@@ -0,0 +1,161 @@
+<script lang="ts">
+  import {
+    Button,
+    ButtonGroup,
+    Dropdown,
+    DropdownItem,
+    Input,
+    Label,
+    Select,
+    Spinner,
+  } from 'flowbite-svelte';
+  import { writable } from 'svelte/store';
+  import * as Realm from 'realm-web';
+  import { realmApp } from '../../main';
+  import { Constants, MongoCollections } from '../../Constants';
+  import { onMount } from 'svelte';
+  import type { UserData } from '../models/UserData/UserData';
+  import {
+    Table,
+    TableBody,
+    TableBodyCell,
+    TableBodyRow,
+    TableHead,
+    TableHeadCell,
+  } from 'flowbite-svelte';
+  import { AccountRole } from '../enums/AccountRole';
+  import { AccountState } from '../enums/AccountState';
+  import { each } from 'svelte/internal';
+
+  let isLoading = true;
+  const {
+    BSON: { ObjectId },
+  } = Realm;
+
+  onMount(async () => {
+    await getUser();
+  });
+
+  var user: UserData;
+
+  async function getUser() {
+    const mongo = realmApp.currentUser.mongoClient(
+      import.meta.env.VITE_DATA_SOURCE_NAME
+    );
+
+    const data = mongo
+      .db(Constants.DatabaseName)
+      .collection(MongoCollections.Users);
+    const result = (await data.find()) as UserData[];
+    user = result.find((user) => user.personalId === realmApp.currentUser.id);
+    isLoading = false;
+  }
+
+  function onSaveAll(e: MouseEvent): void {
+    const mongo = realmApp.currentUser.mongoClient(
+      import.meta.env.VITE_DATA_SOURCE_NAME
+    );
+
+    const data = mongo
+      .db(Constants.DatabaseName)
+      .collection(MongoCollections.Users)
+      .updateOne(
+        { personalId: realmApp.currentUser.id },
```

Comment:
mrkni pak na mou branch, toto je lepsi delat pres cely entity, mas jistotu ze to neposeres a neudelas treba misto
surname => surmame

Reply:
Ok, přepíšu